draft-ietf-masque-connect-ip.md

```diff
@@ -1331,6 +1331,37 @@ and outer IP headers does not apply because the outer connection will react
 correctly to congestion notifications if it uses ECN. The inner traffic can
 also use ECN, independently of whether it is in use on the outer connection.
 
+## Diffserv Considerations
+
+The tunneled IP traffic can have a Diffserv Code Point (DSCP) {{!DSCP=RFC2474}}
+set in the traffic class IP header field to request a particular per hop
+behavior. The client or IP proxy handling an IP proxying request if part of the
+differv domain and configured with the domains mapping MAY implement traffic
+differentiation. However, the use of HTTP can limit the possibilities for
+differentiated treatment of the tunneled IP packets on the path between the
+ingress and egress.
+
+If a client or IP proxy with a connection containing an IP Proxying request
+stream are using congestion control for that stream, independent if datagram or
+capsules are used, all traffic independent of DSCP marking will be treated
+equally within that transport connection. The client or proxy MUST NOT copy the
+DSCP field from the inner IP header to the outer IP header of the packet
+carrying this packet. Instead an application intending to use IP proxying with
+different DSCP will have to establish multiple transport connections, one per
+each DSCP to be used, and requesting proxying of these flows specifically. It
+need to be noted that as this specification does not allow DSCP as a selector
+for the traffic matching the Connect-IP request.
```

Comment:
I can't quite parse this sentence.

Reply:
So I have pushed a text change that reword that to: 

It needs to be noted that the Connect-IP request per current specification does not define a way for a scoped request that use DSCP as a traffic selector.

Is that clearer?